src/components/SqlEditor.tsx

```diff
@@ -1,44 +1,55 @@
 import React, { useState } from 'react';
-import { QueryEditorProps } from '@grafana/data';
+import { CoreApp, QueryEditorProps } from '@grafana/data';
 import { CodeEditor } from '@grafana/ui';
-import { Datasource } from '../data/CHDatasource';
+import { Datasource } from 'data/CHDatasource';
 import { registerSQL, Range, Fetcher } from './sqlProvider';
-import { CHQuery, CHConfig, QueryType, CHSQLQuery } from '../types';
-import { styles } from '../styles';
+import { CHConfig } from 'types/config';
+import { CHQuery, EditorType, CHSqlQuery } from 'types/sql';
+import { styles } from 'styles';
 import { fetchSuggestions as sugg, Schema } from './suggestions';
```

Comment:
nit: Let's avoid unnecessary abbreviations. `fetchSuggestions` is more self-descriptive than `sugg`